### DISCUSSION TITLE
Two inverters not connected to each other, readings seem to be combined over API

### DISCUSSION THREAD

**Discussion by bengrins** [macxq/foxess-ha#252] [Q&A]
I have two inverters, that are not linked. I've tried added them as individual entities, but whatever I try I seem to get combined readings instead of individual as they appear in the portal:

![image](https://github.com/macxq/foxess-ha/assets/114019803/a25a6e3f-8305-4d77-81d3-dce4e30b974c)

Any ideas?

**Comment by FozzieUK** [comment]
Are they in the same user account, i.e. do they share the same apikey ? (If you look at the API management screen and it says 2,880 api calls a day they are in the same account).

If they are in the same account you should have your configuration set up as per the multi-inverter option with the individual DeviceSN’s set up for each - if you have set up a ‘name’ you should then see sensors for each ‘Fox1 - xxxxx’ and ‘Fox2 - xxxxx’

**Comment by bengrins** [comment]
Yes, they are the same account and using the same API key. I've tried this configuration and don't seem to have any luck getting both to show data, the entities all have an error. I'll try it again and see if I simply made a mistake. Thanks!

**Reply by FozzieUK** [reply]
Ok, if you can’t get it running - either post some logs Settings, System, Logs, ‘Load Full logs’ and copy any ‘error’ lines that relate to the foxess component below or email me (b5djf@yahoo.com) the lines from your platform foxess: and i’ll load it up on my test system and see where its going wrong.
